MAKEALL

```diff
@@ -15,7 +15,7 @@ build_config() {
 	deploy_path=$DEPLOY_FOLDER/$folder_name
 	make -j`nproc` O=$build_path D=$deploy_path mrproper
 	make -j`nproc` O=$build_path D=$deploy_path $config
-	make -j`nproc` O=$build_path D=$deploy_path
+	make -j`nproc` O=$build_path D=$deploy_path sdcard
```

Comment:
just keep the previous rule as well.. in case some day sdcard rule just does image alone and does'nt generate other artifacts

Reply:
I don't see us changing behavior such that sdcard would not be dependent upon building the artifacts going on the card.  But I can keep both - note that u-boot isn't completely efficient with rebuilding when nothing changed, so the second make call will be regenerating binaries.  Wasted cycles...